### ISSUE TITLE
Из sizeValidate сразу возвращать интежер

### ISSUE THREAD

**Issue by Znack** [Burize/FULLSTACK-GameOfLife#13]
В ретурне тоже сделать parseInt(size, 10)

**Comment by Burize** [comment]
Это для случая если будет аргумент будет float? Или это не для конкретного варианта, а чтобы в принципе в любом случае вернул int?

**Comment by Znack** [comment]
Да-да, вдруг там что-то неожиданное будет. А это с  js вполне возможно :)
